### ISSUE TITLE
Ignore missing fields on patching

### ISSUE THREAD

**Issue by dorians** [scalalandio/chimney#106]
```
case class Platform(name: String)

case class PlatformConfiguration(name: Option[String] = None,
                                 visible: Option[Boolean] = None)

platform.patchWith(platformConfiguration)

// Field named 'visible' not found in target patching type tech.cryptonomic.conseil.generic.chain.PlatformDiscoveryTypes.Platform!
```

I need an option to ignore or select fields to override.

**Comment by krzemin** [comment]
Possibly similar to #98.

**Comment by krzemin** [comment]
Fixed in #130.
